src/Utils/Ethereum.js

```diff
@@ -163,6 +163,14 @@ export function isValidContractInputParameter(contractInput, value, required = f
  * @returns {object}
  */
 export function parseEthValue(value) {
+    if (value.length <= 2) {
```

Comment:
I believe we shouldn't check the length of the value as this only covers the case when we pass `0x` as a value. If we were to pass just a string `"9"` this would return 0.

As `0x` is not a valid hexadecimal string (same as an empty string) I would possibly change this if statement to check if the value is not set and is not `0x`.

```suggestion
    if (!value || value === '0x') {
```